aiu_fms_testing_utils/utils/encoders.py

```diff
@@ -0,0 +1,646 @@
+# Standard
+from tqdm import tqdm
+import argparse
+import collections
+import json
+import os
+import time
+
+# Third Party
+from datasets import load_dataset
+from fms.models.hf import to_hf_api
+from torch.utils.data import DataLoader
+from transformers import (
+    default_data_collator,
+    DataCollatorWithPadding,
+    EvalPrediction,
+    pipeline,
+    PreTrainedModel,
+    PreTrainedTokenizerBase,
+    RobertaTokenizerFast,
+)
```

Comment:
same here, are we using HF always? or only when model architecture is hf_pretrained? If it's the second case, then the imports shouldn't be here. Plus, the FMS models are not PreTrainedModels